dev-db/docker-compose.yaml

```diff
@@ -1,15 +1,38 @@
 services:
   alpha:
-    image: "scylladb/scylla:5.4.6"
+    image: "scylladb/scylla:2025.4.0-rc3"
+    user: "${UID:-1000}:${GID:-1000}"
+    environment:
+      - HOME=/tmp
     networks:
       scylla_bridge:
         ipv4_address: 172.18.0.2
         aliases:
           - argus-alpha
     command: --smp 1 --overprovisioned 1 --io-setup 0
     volumes:
-      - ./alpha-data:/var/lib/scylla/data:rw
+      - ./scylla:/var/lib/scylla:rw
       - ./alpha-config:/etc/scylla:rw
+      - ./alpha-config.d:/etc/scylla.d:rw
+  vector-store:
```

Comment:
Is this now a critical component of argus? This should probably be mentioned in the commit/readme and docs should be updated.